projects/cal/src/lib/components/kc-cal-day/kc-cal-day.component.scss

```diff
@@ -26,7 +26,6 @@
 
   &--other {
     color: #94a3b8;
-    cursor: default;
```

Comment:
why did you remove it?

Reply:
if you want the user to be able to click on the days of the previous and the following month, then implement in another pull request because there are more cases that need to be implemented

Reply:
to show the pointer cursor instead, you can click on "other" (offsets) days

Reply:
continuation of the above message

1 When you click on a day that does not belong to the current month, then the month must also change
2 Hover will have to work on them too, just with a different shade
3 And also the selected range must be seen but also with a different shade

Reply:
![Peek 2023-12-28 11-45](https://github.com/k5cjs/k5cjs/assets/19764133/7dfa142a-8f41-47b0-850a-f0f322095a16)